__tests__/app.test.js

```diff
@@ -322,6 +322,20 @@ describe("/articles/:article_id/comments", () => {
   });
 });
 
+describe("/comments/:comment_id", () => {
+  test("DELETE:204 deletes the specified comment by comment_id", () => {
+    return req(app).delete("/api/comments/1").expect(204);
```

Comment:
This is absolutely fine, just as a sidenote for some extra food for thought: you *could* technically also chain on another .then where you try to get the comment you just deleted (which should return a 404)